catalog/app/containers/Bucket/PackageUpdateDialog.js

```diff
@@ -1028,23 +1055,18 @@ function DialogForm({
                   required: 'Enter a package name',
                   invalid: 'Invalid package name',
                 }}
-                margin="normal"
-                fullWidth
```

Comment:
i believe we should not encapsulate these layout-related props into the specialized component -- they should be set by the calling component when required

Reply:
Why not?

Now there three levels:
1. RF.Form and props related to final-form
2. CommitInput and NameInput and props related to the visual look
3. Field - an abstract component that can be anything

Reply:
ok for now, we'll see if need to change this going forward